server/plugin/configuration.go

```diff
@@ -219,9 +218,11 @@ func (p *Plugin) OnConfigurationChange() error {
 			enableDiagnostics = *configValue
```

Comment:
I think the code becomes easier to read if the whole block starting in L215 is part of a `if p.tracker != nil {`.

What do you think?